browser/js/content.js

```diff
@@ -30,75 +27,37 @@ function init() {
     }
 
     const user = matchService(window.location, document);
-    if (!user) return;
 
-    chrome.runtime.sendMessage({
-      "method": "passivequery",
-      "to": user.query(),
-    }, function(response) {
-      if (response.status !== "ok") return;
-      user.services["keybase"] = safeHTML(response.result["username"]);
-    });
-  });
+    // Passive queries?
+    if (options["profile-passive-queries"] === true && user) { // undefined defaults to false
+      chrome.runtime.sendMessage({
+        "method": "passivequery",
+        "to": user.query(),
+      }, function(response) {
+        if (response.status !== "ok") return;
+        user.services["keybase"] = safeHTML(response.result["username"]);
+      });
+    }
 
-  // Inject Reddit thread DOM changes?
-  if (location.hostname.endsWith('.reddit.com') && checkThread.test(location.pathname)) {
-    chrome.storage.sync.get("reddit-thread-reply", function(options) {
-      // Is thread replies enabled?
-      if (options["reddit-thread-reply"] === false) return; // Default true
+    // Inject Reddit replies?
+    if (options["reddit-thread-reply"] !== false) { // undefined defaults to true
```

Comment:
Yes, the way the Chrome sync options work, the keys are not present by default so they're undefined). In this case we want reddit-thread-reply stuff to be enabled by default so undefined defaults to true (per the comment next to it).